src/renderer/Generics/redesign/NotificationIcon/NotificationIcon.tsx

```diff
@@ -0,0 +1,82 @@
+import { IconId } from 'renderer/assets/images/icons';
+import {
+  iconBackground,
+  hasStatusStyle,
+  smallStyle,
+  statusStyle,
+  containerStyle,
+} from './notificationIcon.css';
+import { Icon } from '../Icon/Icon';
+import { common } from '../theme.css';
+
+export interface NotificationIconProps {
+  /**
+   * What's the status?
+   */
+  status: 'info' | 'completed' | 'download' | 'warning' | 'error';
+  /**
+   * Is it unread?
+   */
+  unread?: boolean;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const NotificationIcon = ({ status, unread }: NotificationIconProps) => {
+  let statusComponent = null;
+  let unreadStyle = '';
+  if (unread) {
+    statusComponent = <div className={statusStyle} />;
+    unreadStyle = hasStatusStyle;
+  }
+
+  const iconObject = {
+    color: '',
+    backgroundColor: '',
+    iconId: 'infocirclefilled',
+  };
+  switch (status) {
+    case 'info':
+      iconObject.color = common.color.green500;
+      iconObject.backgroundColor = 'rgba(18, 186, 108, 0.08)';
+      iconObject.iconId = 'infocirclefilled';
+      break;
+    case 'completed':
+      iconObject.color = common.color.green500;
+      iconObject.backgroundColor = 'rgba(18, 186, 108, 0.08)';
+      iconObject.iconId = 'checkcirclefilled';
+      break;
+    case 'download':
+      iconObject.color = common.color.blue500;
+      iconObject.backgroundColor = 'rgba(19, 122, 248, 0.08)';
+      iconObject.iconId = 'download1';
+      break;
+    case 'warning':
+      iconObject.color = common.color.orange400;
+      iconObject.backgroundColor = 'rgba(247, 144, 9, 0.12)';
+      iconObject.iconId = 'warningcirclefilled';
+      break;
+    default:
+  }
+
+  return (
+    <div className={containerStyle}>
+      {/* https://stackoverflow.com/questions/6040005/relatively-position-an-element-without-it-taking-up-space-in-document-flow */}
+      <div style={{ position: 'relative', width: 0, height: 0 }}>
```

Comment:
This is solved in the modalmanager branch because the modals are now at the top of the html, and the z-index for the modal is higher than the elements in Notifications view.

<img width="1196" alt="Screen Shot 2023-02-05 at 2 11 22 PM" src="https://user-images.githubusercontent.com/99524051/216848997-914465b4-7503-415a-a7eb-34000acbf01a.png">
<img width="1047" alt="Screen Shot 2023-02-05 at 2 13 44 PM" src="https://user-images.githubusercontent.com/99524051/216849001-ab94138b-746d-48cd-aa0a-f90a3479cde0.png">

Reply:
Should I still make changes for this PR? (notifications aren't being added into localStorage just yet though, so it shouldn't be seen by users yet)